quickjs/common/java/app/cash/quickjs/QuickJs.java

```diff
@@ -150,6 +150,27 @@ public String toString() {
     return (T) proxy;
   }
 
+  /**
+   * Compile {@code sourceCode} and return the bytecode. {@code fileName} will be used in error
+   * reporting.
+   *
+   * Returns null if the sourceCode could not be compiled.
```

Comment:
```suggestion
   * @return null if the sourceCode could not be compiled.
```
Otherwise your double newline doesn't do anything and it all gets slammed into a single paragraph in the javadoc.

Also does this mean you get no information on syntax errors like you would with calling `evaluate`?

Reply:
Good point, I took a look and we do get an exception we can throw instead.  Will switch to that, but it's not immediately clear how to get line numbers out for the failed compile (adding a TODO).